### ISSUE TITLE
Check migrations

### ISSUE THREAD

**Issue by ihorml** [liquidnft/lnft#47]
Running the latest `master` with **fresh install** I can see the following errors:

<img width="354" alt="Screenshot 2021-10-05 at 12 23 06" src="https://user-images.githubusercontent.com/7770343/135996662-fd065e38-972d-4942-9046-c61c6508aa03.png">
<img width="328" alt="Screenshot 2021-10-05 at 12 23 24" src="https://user-images.githubusercontent.com/7770343/135996687-59cf8fb9-7e54-4ac1-9454-4c03eb3e2906.png">

**Comment by ihorml** [comment]
@asoltys after login to Hasura admin, I click on some fix button (I don't remember proper naming of it) and it works.

**Comment by Eskyee** [comment]
Yes, @asoltys  has now solved this bug , if you update to latest version.
you can close this if nothing else.
